ZTP.md

```diff
@@ -65,28 +65,30 @@ Should there be a ztp setting that automatically goes back into ZTP mode if the
-|     224     | snmp_community              | snmpcommunity DHCP hook updates /etc/sonic/snmp.yml file with  provided value |
-|     225     | minigraph_url               | graphserviceurl DHCP hook updates the file /tmp/dhcp_graph_url with the provided url. updategraph service processes uses it for further processing. |
-|     226     | acl_url                     | graphserviceurl DHCP hook updates the file /tmp/dhcp_acl_url with the provided url. updategraph service processes uses it for further processing. |
-|    239      | ztp_provisioning_script_url    | URL for a script which needs to be downloaded and executed by ZTP service on the switch. |
-
-The following are the DHCPv6 options used by the SONiC switch to receive input provisioning data for ZTP service.
-
-| DHCPv6 Option | Name         | Description                                       |
-|:-----------:|:-------------------|:-------------------------------------------------|
-| 15 | user-class | Used to optionally identify the type or category of user or    applications it represents. SONiC switches set this value to  "SONiC-ZTP". |
-|    59    | boot-file-url | URL to download the ZTP JSON file. |
-|    239      | ztp_provisioning_script_url    | URL for a script which needs to be downloaded and executed by ZTP service on the switch. |
+The following are the DHCPv4 options used for Network Boot and Secure ZTP
+
+|   DHCP Option | Name                 | Description                                                                                                                |
+| :-----------: | :------------------- | :-------------------------------------------------                                                                         |
+|            60 | class-identifier     | Class Identifier (RFC 2132) (PXE) (HTTP(s)) (i.e. "PXEClient:Arch:xxxxx:UNDI:yyyzzz", "HTTPClient:Arch:xxxxx:UNDI:yyyzzz") |
+|            61 | client-identifier    | unique identifier (RFC 2132, RFC 4578) (PXE) (HTTP(s))                                                                     |
+|            66 | Server-Name          | TFTP Server Name (RFC 2132) (PXE)                                                                                          |
+|            67 | Bootfile-Name        | Boot File Name (RFC 2132) (PXE) or Boot URI (HTTP(s))                                                                      |
+|            77 | user-class           | User Class Information (RFC 3004)(i.e. "iPXE", "SONiC-ZTP")                                                                |
+|            93 | Client-System        | Client System Architecture (RFC 4578) (PXE) (IA x86 PC, x86 EFI boot, x64 EFI boot)                                        |
+|            93 | Client-NDI           | Client Network Device Interface (RFC 4578) (PXE)                                                                           |
```

Comment:
Appears you've enumerated option 93 twice here?

Reply:
You are correct the second one should have been 94

https://www.iana.org/assignments/bootp-dhcp-parameters/bootp-dhcp-parameters.xml